docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc

```diff
@@ -52,6 +52,8 @@ include::GettingStarted.adoc[]
 
 include::BringingUpMatterNodeForCertificationTesting.adoc[]
 
+include::BringingUpTheMatterPythonREPL.adoc[]
```

Comment:
The file name here is not the same as the actual file.

```suggestion
include::BringingUpMatterPythonREPL.adoc[]
```

Reply:
Using `BringingUp` on the new section was on purpose because it's kind of related to the other one.
`BringingUpMatterNodeForCertificationTesting.adoc` talks about using the sample apps and `BringingUpTheMatterPythonREPL.adoc` talks about using `chip-repl`, which are all located in the same path in the host.